pallets/runtime/common/src/bridge.rs

```diff
@@ -254,16 +254,32 @@ decl_storage! {
             }
             let creator_key = AccountKey::try_from(config.creator.clone().encode()).expect("cannot create the bridge creator account");
             let creator_did = Context::current_identity_or::<identity::Module<T>>(&creator_key).expect("bridge creator account has no identity");
-
+            <balances::Module<T>>::top_up_identity_balance(
```

Comment:
I don't think we should be topping up any identity balance at genesis. If we do, it should be done in the balances module and amount should be configurable in chain spec.

Reply:
I don't mind if there are no identity balance transactions at genesis. The only reason for it here is paying fees for accepting bridge multisig authorizations. I can't think of any reason to have more general identity transactions at genesis in the balances module.

Reply:
The reason I want all poly balances to be defined in a general format is because we can not create POLYX from thin air, all of the POLYX in the system at genesis must map 1 is to 1 with POLY locked on Ethereum. Having a general format makes accounting easier.

ps If the auths are accepted at genesis, they do not pay any fee. If they are accepted post genesis, we can transfer some Poly to the identity manually before accepting auths.

Reply:
> The reason I want all poly balances to be defined in a general format is because we can not create POLYX from thin air, all of the POLYX in the system at genesis must map 1 is to 1 with POLY locked on Ethereum. Having a general format makes accounting easier.

Makes sense.

> ps If the auths are accepted at genesis, they do not pay any fee. If they are accepted post genesis, we can transfer some Poly to the identity manually before accepting auths.

Without balance in the identity I was getting `InsufficientBalance` at genesis.

Reply:
If we want the relayers to work out of the box on dev chains, then I believe Alice's identity needs POLYX. @maxsam4 my reading of fee_details.rs#L106 is that it will always be the identity that pays for multisig / bridge txs.

I agree that any balance changes should be configurable through chain_spec - my suggestion would be to extend the endowed account logic to also allow us to endow identities in a similar way.

Reply:
> I agree that any balance changes should be configurable through chain_spec - my suggestion would be to extend the endowed account logic to also allow us to endow identities in a similar way.

@adamdossa please let me know if you are happy with the changes I made in `chain_spec.rs`. Alice's identity is endowed only on dev chains.